changelog.txt

```diff
@@ -1,10 +1,14 @@
 0.88.0
  * BREAKING CHANGE: DynamicRealm.executeTransaction() now directly throws any RuntimeException instead of wrapping it in a RealmException (#1682).
  * BREAKING CHANGE: DynamicRealm.executeTransaction() now throws IllegalArgumentException instead of silently accepting a null Transaction object.
+ * BREAKING CHANGE: RealmQuery is now final.
  * Fixed an error occurring during test and connectedCheck of unit test example (#1934).
  * Added RealmQuery.isNotEmpty() (#2025). (Thank you @stk1m1)
  * Improved .so loading by using ReLinker (https://github.com/KeepSafe/ReLinker).
  * Improved performance of RealmList#contains() (#897).
+ * Added support for RealmQuery.asObservable().
+ * RealmQuery now implements Cloneable.
+ * Added Realm.threadLocalVersion(RealmQuery).
```

Comment:
How about rearranging items.

  * Added RealmQuery.isNotEmpty() (#2025). (Thank you @stk1m1)
  * Added support for RealmQuery.asObservable().
  * Added Realm.threadLocalVersion(RealmQuery).
  * Improved .so loading by using ReLinker (https://github.com/KeepSafe/ReLinker).
  * Improved performance of RealmList#contains() (#897).
  * RealmQuery now implements Cloneable.